wolfcrypt/test/test.c

```diff
@@ -713,8 +713,29 @@ WOLFSSL_TEST_SUBROUTINE wc_test_ret_t aes_eax_test(void);
 /* General big buffer size for many tests. */
 #define FOURK_BUF 4096
 
+#if defined(WOLFSSL_ESPIDF)
+    static const char* TAG = "wolfcrypt_test"; /* ESP_LOG() breadcrumb */
```

Comment:
Is this only used when WOLFSSL_ESPIDF_ERROR_PAUSE is defined? If so please move it down into that macro guarded section of code to avoid unused warnings.

Reply:
No, it wasn't the only place, but I cleaned up the other one that was hard-coded with "wolfcrypt_test" text. 

There will be a separate, future exercise to replace references to `printf` with the Espressif `ESP_LOG`.

Reply:
Sounds good, when replacing the printf's I recommend following the existing XPRINTF override pattern we have for using a custom printf in the tests.